packages/@aws-cdk/aws-eks/test/test.fargate.ts

```diff
@@ -185,5 +185,38 @@ export = {
       }
     }));
     test.done();
+  },
+
+  'can create FargateCluster with a custom profile'(test: Test) {
+    // GIVEN
+    const stack = new Stack();
+
+    // WHEN
+    new eks.FargateCluster(stack, 'FargateCluster', {
+      fargateProfile: {
+        fargateProfileName: 'custom', selectors: [{namespace: 'foo'}, {namespace: 'bar'}]
```

Comment:
Can you please add a unit test that covers the case where fargateProfileName is not specified?

Reply:
@eladb Sure. Should I drop the `fargate` prefix as well given [this comment](https://github.com/aws/aws-cdk/pull/6074#discussion_r374248832)?

Reply:
Added